### PR TITLE
Sponsor_FAQ section

### LINKED ISSUE #8: Landing Page - Section 6 - Want to be a sponsor & FAQ section
**Is your feature request related to a problem? Please describe.**
No, its the initial website building part.

**Describe the solution you'd like**
Section-6 of the Landing Page:
* Button about us in section 2 issue is build. Try to take it from there if built, or make your own.
* Replicate the chat FAQ as close as possible.
* Make sure that the FAQs are responsive for mobile version as well.

**Describe alternatives you've considered**
Discuss the design alternatives with designers and project maintainers.

**Additional context**
Prototype: https://www.figma.com/proto/4a8ZglxR98YOEQpmXu93Ng/Dhruva-team-library?page-id=0%3A1&node-id=312%3A18&viewport=241%2C48%2C0.2&scaling=min-zoom

### PR DESCRIPTION
**Pull request by ishaan5199**
# Description

Added the Sponsor_FAQ component based on the design provided in the figma prototype

Dependencies : Nothing extra

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Browsers Tested on?

- [x] Chrome
- [x] Firefox
- [x] Edge

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have checked the responsiveness of the component I built
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

File: components/Sponsor_Faq/FAQ.module.css [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    z-index: -1;
+}
+.SponsorBack{
+    width: 100%;
+    height: 100%;
+    background-color: #181414;
```
Comment (charles-vj) [inline components/Sponsor_Faq/FAQ.module.css]: - This background should be completely black. Do use the variable for this.

File: components/Sponsor_Faq/FAQ.module.css [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+    transform: translate(0.25em, 0.25em);
+}
+
+.chat{
+    margin-top: 3rem;  
+    padding: .95rem; 
+    font-family: var(--coming);
+    font-size: larger;
+}
+.chatW{
```
Comment (charles-vj) [inline components/Sponsor_Faq/FAQ.module.css:106]: - Reduce the width of the white chat bubble as given in the figma file

**Review by charles-vj — CHANGES_REQUESTED** [review]
Just some minor changes needed

File: components/Sponsor_Faq/FAQ.module.css (hunk 2 above)
Comment (GetPsyched6) [reply components/Sponsor_Faq/FAQ.module.css:106]: Made the changes required

**Review by joshi008 — CHANGES_REQUESTED** [review]
Kindly resolve the conflicts and while giving the pull request  don't include package-lock.json . 

Secondly in the mobile version of the website, the width of the chat FAQ could be increased to full width, as right now it is increasing its height to compensate the increase in text. 
image

**Comment by faiz-hasan11** [comment]
Kindly take a pull  from latest development branch , merge your code locally and then push

**Comment by faiz-hasan11** [comment]
Your branch is behind 8 commits , kindly pull latest code from development branch , merge your code locally and then push . I wont repeat again.

**Comment by GetPsyched6** [comment]
> Your branch is behind 8 commits , kindly pull latest code from development branch , merge your code locally and then push . I wont repeat again.

Whoops my mistake, will merge and push

**Pull request closed without merging**